.github/workflows/close-stale-issues.yml

```diff
@@ -0,0 +1,37 @@
+name: stale
+
+on:
+  schedule:
+    - cron: '30 1 * * *'
+
+permissions:
+  issues: write
+  pull-requests: write
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      -
+        name: Run
+        uses: actions/stale@v9
```

Comment:
Looking at this action it would be good to handle `debug-only` to check what the action would do first: https://github.com/actions/stale/blob/3f3b0175e8c66fb49b9a6d5a0cd1f8436d4c3ab6/action.yml#L131-L134

And related to previous comment we could add an input to the `workflow_dispatch` event named `dry-run`. Would be similar to: https://github.com/docker/packaging/blob/3ac603d1bfbf4f1e347c2a35e75585afae9d7276/.github/workflows/release-buildx.yml#L18-L21

![image](https://github.com/docker/buildx/assets/1951866/adbbc165-b584-4043-867d-8f7448b470c6)

I can push an extra commit for this if you want.

Reply:
Go for it :flex